FlowCal/excel_ui.py

```diff
@@ -133,7 +141,8 @@ def read_table(filename, sheetname, index_col=None):
 
     """
     # Catch sheetname as list or None
-    if sheetname is None or hasattr(sheetname, '__iter__'):
+    if sheetname is None or \
+        (hasattr(sheetname, '__iter__') and not isinstance(sheetname, str)):
```

Comment:
Another option would be to do the following:

```python
if sys.version_info[0]==3:
    basestring = str

if isinstance(s, basestring):
    # do whatever
```

Since `basestring` is not defined in python3, this would work seamlessly.